src/type.zig

```diff
@@ -1530,15 +1530,15 @@ pub const Type = extern union {
             .const_slice,
             .mut_slice,
             => {
-                if (self.elemType().hasCodeGenBits()) return @divExact(target.cpu.arch.ptrBitWidth(), 8) * 2;
+                if (!self.elemType().hasCodeGenBits()) return @divExact(target.cpu.arch.ptrBitWidth(), 8) * 2;
```

Comment:
This looks wrong, if the element type is non-zero sized you have both a pointer and a length field, notice the `* 2` at the end.

Reply:
Ah, for the other ones I think it was a typo, but for this one it was not. Thanks